modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,58 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+
+== Add Support for Apple Universal Links
+
+There's some special changes that need to be made. 
+Quoting from Apple's official documentation on {apple-universal-link-support-url}[Universal Link Support]:
+
+[quote]
+____
+Adding support for universal links is easy. 
+There are three steps you need to take:
+
+. Create {apple-app-site-association-file-url}[an apple-app-site-association file] that contains JSON data about the URLs that your app can handle.
+. Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the `.well-known` subdirectory.
+. Prepare your app to handle universal links. 
+____
+
+The `apple-app-site-association` will be generated by xref:branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app[ownBrander], and needs to be served statically on your site through HTTPS. 
```

Comment:
...
The `apple-app-site-association` is a subdirectory of your owncloud URL. Data generated by xref:branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app[ownBrander] will be accessed via this subdirectory and needs to be served statically on your site through HTTPS. `apple-app-site-association` can be either a direct subdirectory or `/.well-known/apple-app-site-association`. The name `apple-app-site-association` is given and must not be changed. The file which gets accessed when using this subdirectory is also named `apple-app-site-association` without any extension.
When this subdirectory is accessed, the webserver must be instructed that the content type served is defined as `'application/json'`.
The physical path which is used when accessing this directory must be defined in your webserver config.
NOTE: Please consider when using a physical path to the file inside your owncloud directory, that this file must be present when you upgrade ownCloud or you chose a different path outside owncloud.
...

I hope this makes it more clear what it is about.
@settermjd pls improve/reword if needed.

Reply:
So this text is all that's required? I didn't touch the text much, as I know very little about the subject matter.

Reply:
That text should be added, pls check for writung improvements.

Reply:
I've added it in and made some adjustments. However, it feels rough to me. Can you review pls, @mmattel?